test/util/IdTableHelpers.h

```diff
@@ -57,6 +57,47 @@ class CopyableIdTable : public TableImpl<N> {
 using IntOrId = std::variant<int64_t, Id>;
 using VectorTable = std::vector<std::vector<IntOrId>>;
 
+// Helper: construct a VectorTable from nested initializer lists of integers.
+static inline VectorTable makeVectorTable(
+    std::initializer_list<std::initializer_list<size_t>> rows) {
+  VectorTable vt;
+  for (auto rowIl : rows) {
+    VectorTable::value_type row;
+    for (auto v : rowIl) {
+      row.push_back(IntOrId(static_cast<int64_t>(v)));
+    }
+    vt.push_back(std::move(row));
+  }
+  return vt;
+}
```

Comment:
I am not sure whether this function is necessary, can't you just do `VectorTable{{3, 4}, {7, 8}, {103, 102}};` directly, which is equivalent to this function?